samples/ImplicitFlow/AuthorizationServer/Properties/launchSettings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:54540",
```

Comment:
It's a pain that there are two launchSettings.json files, one for VS Code and one for Visual Studio, eh.